cmd/snap-update-ns/utils.go

```diff
@@ -339,6 +339,148 @@ func MkSymlink(dirFd int, dirName string, name string, oldname string, rs *Restr
 	return nil
 }
 
+// FileInWayError indicates that a directory cannot be created because
+// a file with the same name exists. This error is intended to be used
+// with MkdirAllWithin.
+type FileInWayError struct {
+	File string
```

Comment:
usually File are os.File  so probably should have a different name like Path or Filename

Reply:
Fixed